### PR TITLE
Views

### PR DESCRIPTION
**Pull request by Tumworobere**
**In this project, I achieved the following:**

   - Implemented the design from the sneak peek wireframes
   - Made sure everything functions good.
   - Used the custom created methods.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -1,5 +1,11 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @posts_list = @user.recent_posts
+  end
```
Comment (AshakaE) [inline app/controllers/posts_controller.rb]: The Index method is supposed to show all users, this allows current user to all other user's posts.

- [ ] Kindly update index method

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,31 @@
...
+    <div class='post'>
+      <h3>Post # <%= u.id %></h3>
+      <p> Text:<%= u.Text %></p>
+      <div>
+        Comments: <%= u.comments_counter %>
+        Likes: <%= u.likes_counter %>
+      </div>
+    </div>
+    <% end %>
+  </ul>
```
Comment (AshakaE) [inline app/views/users/show.html.erb]: - [ ] Kindly update action view `show` to display recent posts for the current user

**Review by AshakaE — CHANGES_REQUESTED** [review]
Hi @Tumworobere  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @Tumworobere, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Tumworobere** [comment]
> Hi 👋 @Tumworobere,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thank you @elisha2kyakpo1

**Pull request merged**